### PR TITLE
[IMP] stock: Tick checkbox for refund by default in reverse transfer

### PR DESCRIPTION
**Pull request by hdh-odoo**
**Description of the issue/feature this PR addresses**: 
- Task: https://www.odoo.com/web#id=1962183&action=333&active_id=131&model=project.task&view_type=form&menu_id=4720
- Pad: https://pad.odoo.com/p/r.61bcd8c2452ff710300a6e82db02e59e

**Current behavior before PR:** 
- The checkbox of to_refund isn't ticked by default.

**Desired behavior after PR is merged:**
- The checkbox of to_refund will be ticked by default.

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/stock/wizard/stock_picking_return.py [hunk 1]
```diff
@@ -59,8 +59,14 @@ def _onchange_picking_id(self):
...
             quantity = float_round(quantity, precision_rounding=move.product_uom.rounding)
-            product_return_moves.append((0, 0, {'product_id': move.product_id.id, 'quantity': quantity, 'move_id': move.id, 'uom_id': move.product_id.uom_id.id}))
-
+            values = self.product_return_moves.default_get(self.product_return_moves.fields_get())
+            values.update({
+                'product_id': move.product_id.id,
+                'quantity': quantity,
+                'move_id': move.id,
+                'uom_id': move.product_id.uom_id.id
+            })
```
Comment (sle-odoo) [inline addons/stock/wizard/stock_picking_return.py]: why this change?

Comment (hdh-odoo) [reply addons/stock/wizard/stock_picking_return.py]: because we don't have the field "to_refund" in stock module. it'll be coming from stock_account. hence we had to use default_get() to get its default value to use here.

**Comment by sle-odoo** [comment]
robodoo r+

**Comment by cvinh** [comment]
Hello
I think that commit disappeared in last master since stock_account has been refactored
